internal/ignition/ignition_test.go

```diff
@@ -1017,7 +1016,7 @@ var _ = Describe("IgnitionBuilder", func() {
 
 	It("auth_disabled_no_pull_secret_token", func() {
 		mockMirrorRegistriesConfigBuilder.EXPECT().IsMirrorRegistriesConfigured().Return(false).Times(1)
-		text, err := builder.FormatDiscoveryIgnitionFile(context.Background(), &infraEnv, IgnitionConfig{}, false, auth.TypeNone)
+		text, err := builder.FormatDiscoveryIgnitionFile(context.Background(), &infraEnv, IgnitionConfig{}, false, auth.TypeNone, "")
```

Comment:
@paul-maidment maybe I'm missing something here; Why are these test changes?

Reply:
FormatDiscoveryIgnitionFile now expects the parameter discovery_iso_type which has been added to the signature, the test needed to be updated to match the signature change.

Reply:
Allow me to better phrase what I don't understand here: Why do all tests pass `""` as a value for `overrideDiscoveryISOType` instead of some actual values?

Reply:
discovery_iso_type is not mandatory in the API and when it is not present (I think) it will be interpreted as an empty string, I did originally accept a pointer but I was discouraged from this in other feedback.